webpack.config.js

```diff
@@ -1,62 +1,3 @@
-const webpack = require('webpack')
-const path = require('path')
-
-const defaultConfig = {
-  module: {
-    rules: [
-      {
-        test: /\.js$/,
-        use: {
-          loader: 'babel-loader',
-          options: {
-            presets: ['@babel/preset-react', '@babel/preset-env'],
-            plugins: ['@babel/plugin-proposal-class-properties'],
-          },
-        },
-      },
-      {
-        test: /\.css$/,
-        use: ['style-loader', 'css-loader'],
-      },
-      {
-        test: [/\.bmp$/, /\.gif$/, /\.jpe?g$/, /\.png$/, /\.otf$/],
-        loader: require.resolve('url-loader'),
-        options: {
-          // limit: 10000,
-          name: 'static/media/[name].[hash:8].[ext]',
-        },
-      },
-      {
-        test: /\.ttf$/,
-        loader: 'url-loader', // or directly file-loader
-        include: path.resolve(
-          __dirname,
-          'node_modules/react-native-vector-icons'
-        ),
-      },
-    ],
-  },
-  externals: {
-    react: 'React',
-    'react-native': 'ReactNative',
-  },
-  resolve: {
-    modules: ['node_modules'],
-    extensions: [
-      '.web.js',
-      '.js',
-      '.json',
-      '.web.jsx',
-      '.jsx',
-      '.web.react.js',
-      '.react.js',
-    ],
-  },
-  plugins: [
-    new webpack.optimize.LimitChunkCountPlugin({
-      maxChunks: 1,
-    }),
-  ],
-}
+const { defaultConfig } = require('@adalo/cli/src/webpackConfig')
```

Comment:
does using the default from https://github.com/AdaloHQ/cli/blob/main/src/webpackConfig.js work, given that there's no mention of `node_modules/react-native-vector-icons` in it?

Reply:
or will webpack include handling for the icons without an explicit config?

Reply:
It does work, doesn't seem like we don't need webpack to handle that library. My understanding is that it works out of the box since it's installed.

Reply:
Got it, thanks!  FWIW, when built natively, the raw Material Components source code is used as opposed to the runtime builds, so I think we're OK there!